operations/mimir-mixin/alerts/alerts.libsonnet

```diff
@@ -466,6 +466,25 @@ local utils = import 'mixin-utils/utils.libsonnet';
             ||| % $._config,
           },
         },
+        {
+          alert: $.alertName('IngesterReachingInflightPushRequestLimit'),
+          expr: |||
+            (
+                (cortex_ingester_inflight_push_requests_summary{quantile="1.0"} / ignoring(limit, quantile) cortex_ingester_instance_limits{limit="max_inflight_push_requests"})
```

Comment:
Using quantile=1.0 will result in very flaky alerts. I would suggest using lower number.